plugin/trino-raptor-legacy/src/main/java/io/trino/plugin/raptor/legacy/backup/BackupModule.java

```diff
@@ -40,11 +40,11 @@
 
     public BackupModule(Map<String, Module> providers)
     {
-        this.providers = ImmutableMap.<String, Module>builder()
+        this.providers = ImmutableMap.<String, java.lang.Module>builder()
```

Comment:
This is a bad change done by structural replace in the IDE.
The `Module` used here is `com.google.inject.Module`, not `java.lang.Module`.